nixos/alsa: disable OSSEmulation

bb2020 opened the pull request:
###### Motivation for this change
#54074

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

worldofpeace commented:
Can you put the motivation into the commit message? I also think we could do with a release note.

bb2020 commented:
Done

raboof reviewed (commented):
This makes sense to me, but I guess the release note should be moved from `2009` to `2103` now

bb2020 commented:
Right. Updated it now.

bb2020 committed:
nixos/alsa: disable OSSEmulation by default

OSS Emulation is considered incomplete so disabling it by default.
Using user level alsa-oss library (nix-env -iA nixos.alsaOss) over
this kernel module is recommended.

bb2020 commented:
Any update on this getting merged?

worldofpeace commented:
Thank you for your patience :sparkles:

bb2020 commented:
Thanks!
